common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -414,6 +414,12 @@ div.problem {
     }
   }
 
+  div.inline {
+    > span {
```

Comment:
@jolyonb - like I said @frrrances used my computer to make that comment. I will let her answer your questions!